apps/fronttoback/package.json

```diff
@@ -18,6 +18,7 @@
     "@sanity/client": "^3.3.3",
     "@skillrecordings/analytics": "workspace:*",
     "@skillrecordings/convertkit": "workspace:*",
+    "@skillrecordings/database": "workspace:*",
```

Comment:
because sites like EM work fine with skill api, one consideration was that the database package wasn't installed